jep/1/README.adoc

```diff
@@ -256,33 +256,50 @@ let's take a high-level look at how JEP might go.
 
 . **<<submission, Submission>>** - Andrea writes up the proposal using the JEP document template as a guide.
   She includes supporting documentation
-  and a prototype implementation sufficient to convey the viability of the design.
+  and a minimal prototype implementation sufficient to convey the viability of the design.
   She submits the JEP to the JEP editors for
   <<approval, approval as a Draft JEP>>.
   An editor checks the submission and determines it is ready to considered as a JEP.
   They "approve" the submission, assigning the JEP a number, and the submission becomes a "Draft" JEP.
```

Comment:
Should link to the Draft state, I missed this the first few times I wondered where it says how to achieve Draft status.